### ISSUE TITLE
should support span id

### ISSUE THREAD

**Issue by RujulGandhi** [MohamedRejeb/Compose-Rich-Editor#261]
I've encountered an issue with a library I'm using. When I set a unique identifier (id) for a span element in the HTML using `state.setHtml()`, it doesn't seem to retain the id when I retrieve the HTML using `state.toHtml()`.

Here's an example:

```javascript
state.setHtml(`<span id="${username}" style="color: rgba(0, 0, 255, 1.0);">@${username}</span>`);
```

However, when I retrieve the HTML using `state.toHtml()`, the id is missing:

```javascript
state.toHtml(); // Output: <span style="color: rgba(0, 0, 255, 1.0);">@${username}</span>
```

I need to send a unique identifier with every span element so that I can retrieve it later. How can I achieve this?

Thank you.

**Comment by MohamedRejeb** [comment]
Hi,
Thanks for opening this issue, for now retuned html format only contains styles and attributes recognized by the library.
Maybe we can add support for ids at least for now. You need this to have a proper mentions support right? Maybe supporting mentions in the library can fix this.

**Comment by RujulGandhi** [comment]
Yes, I want mention functionality, but how can I achieve id part for now ?

**Comment by RujulGandhi** [comment]
Is there any alternative which you can think to solve this ?
